### PR TITLE
Cleanup installation docs

### PR DESCRIPTION
**Pull request by amagana3**
This PR updates our [local installation](https://registry.terraform.io/providers/Cox-Automotive/alks/latest/docs/guides/local_installation) docs. 

Changes for 0.12 instructions:
- Added a one-liner command. Will grab the latest version, and place it in the `/.terraform.d/plugins` with the appropriate version constraint ON the file name. (previously missing that).
- Added instructions to the 0.12 installation for manually downloading as well as HOW to configure the provider itself.

Changes for 0.13 instructions:
- Added a one-liner command. Will grab the latest version, and place it in the `/.terraform.d/plugins/Cox-Automotive/engineering-enablement/alks/<<LATEST>>/darwin_amd64` directory. Right now the version is hardcoded, so our WIKI has to be updated to include instructions for updating provider versions for the 0.13+ installation instructions.
- Updated some verbiage to be more clear.
- Updated the directory PATH. No longer using `.../plugins/coxautomotive.inc/engineering-enablement/...` as too many people complained that this was far too different than what our registry path is: `"Cox-Automotive/alks"` Note: This only changes the local installation path, not the registry one.

Added a new version matrix / table?
- Basically this tells users which version of Terraform supports which version of ALKS TFP. It's really more used for ancient versions, but anything after `1.3.0` will be supported by all newer versions of TF.

### CODE REVIEW

**Review by ntangy — APPROVED** [review]
LGTM, hopefully it clarifies things for folks out
